doc/RECIPES.md

```diff
@@ -101,3 +141,31 @@ class MyElement extends XElement {
   }
 }
 ```
+
+## How do I add escaped unicode characters in my templates (\u2026)?
```

Comment:
Adds explainer for writing special characters as _html_ versus with JavaScript escapes.